Sprint-1/errors/3.js

```diff
@@ -1,9 +1,14 @@
 const cardNumber = 4533787178994213;
-const last4Digits = cardNumber.slice(-4);
+const last4Digits = cardNumber.slice(4);
```

Comment:
Your code is still calling `.slice()` on a number type value. Have you tried executing your code?